content/en/synthetics/guide/synthetic-test-monitors.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Understanding Synthetic Tests and Alerting Monitors
+kind: guide
+description: Learn about Synthetic monitors created with your Synthetic tests. 
+further_reading:
+- link: "/monitors/manage/"
+  tag: "Documentation"
+  text: "Manage Monitors"
+- link: "/synthetics/metrics/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Metrics"
+---
+
+## Overview
+
+When you create a Synthetic test, Datadog automatically creates an associated monitor. The monitor's alerting conditions in your test configuration impact how you are notified when a Synthetic test moves into an alerting state.
+
+{{< img src="synthetics/guide/synthetics_test_monitors/synthetic_test_monitor.png" alt="Synthetic test monitor" style="width:100%;">}}
+
+This guide describes how Synthetic test monitors behave, how to use a monitor's conditional variables in a Synthetic test configuration, and how to set downtimes and mute monitors.
```

Comment:
Might need to revamp this sentence slightly? It does not seem to comprehensively reflect the content of the guide + it mentions downtimes that I don't see mentioned below?